README.md

```diff
@@ -64,6 +64,38 @@ Before running the app, make sure you have the Bedrock API running whether on lo
 | `BEDROCK_API`         | https://localhost:2300 | Base url of where the API is running |
 | `BEDROCK_API_VERSION` | 1                      | API Version                          |
 
+
+**Run app in dev** 
+`flutter run lib/env/main_dev.dart`  
```

Comment:
This should be `flutter run lib/src/env/main_dev.dart`. I believe the rest are missing the `/src` as well